app/scripts/modules/core/src/pipeline/pipeline.states.ts

```diff
@@ -55,6 +55,14 @@ module(PIPELINE_STATES, [APPLICATION_STATE_PROVIDER]).config(
       views: {
         pipelines: { component: SingleExecutionDetails, $type: 'react' },
       },
+      params: {
+        senderApplication: {
```

Comment:
the name `sender` seems ambigious to me, how about something like `executionParams`